README.md

```diff
@@ -25,3 +25,31 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+Recall for merge sort, the time complexity is $T(n) = \begin{cases} 1 \; is n \le 1 \\ 2T(n/2) + n \; if n > 1$. 
+
+1. For the divide-and-conquer sum algorithm if the array has 0,1, or 2 elements, it will add them up so the time complexity is 1. $T(1) = T(2) = 1$.
+2. Split the array into 3 approximately equal sized thirds. $1$
+3. Sum each halve recursively using mergesum. $3 T(n/3)$
+4. Merge the sums $n$
+
+Thus our recurrence relation is  $T(n) = \begin{cases} 1 \; is n \le 2 \\ 3T(n/3) + n \; if n > 2$.
+
+Now to solve the recurrence relation by substitution:
```

Comment:
The recurrence relation is not readable as it should be. Please use the appropriate notations as the line that start with "Recall for merge sort, ....